tensorflow/training/docker/2.9/py3/cu112/Dockerfile.gpu

```diff
@@ -285,3 +286,116 @@ RUN HOME_DIR=/root \
 RUN rm -rf /tmp/*
 
 CMD ["/bin/bash"]
+
+#################################################################
```

Comment:
E3 recipe is above in the same file. This does not look right.

Reply:
@pinaraws this is a multistage dockerfile. However, it looks like we are using the same framework binary for e3 and sagemaker - is this expected @nish21?

Reply:
Discussed offline on why they're the same.